network-topology/nodes/sv-node/chronicle-release.md

```diff
@@ -13,16 +13,16 @@ The changes introduced in the Chronicle release are detailed in the sections bel
+* **Transaction Digest Algorithms**: The BSV Blockchain will now support both the Original Transaction Digest Algorithm (OTDA) and the BIP143 digest algorithm (with SIGHASH\_FORKID), ensuring compatibility and flexibility for developers and users. This restores the original Bitcoin transaction digest algorithm, enabling developers to have greater flexibilty in utilizing Bitcoin Script.
+* **Selective Malleability Restrictions:** To address concerns about the OTDA's reintroduction of sources of transaction malleability, the application of malleability restrictions will depend on the interaction of the FORKID and RELAX sighash bits. Transactions signed without SIGHASH\_FORKID (or with RELAX enabled) will allow relaxed rules, removing strict enforcement of malleability-related constraints. This flexibility is agnostic to the number of signatures in a transaction.
+* **Business Impact and Flexibility:** In line with the BSV Blockchain's commitment to stability, existing users and applications using the BIP143 digest (with SIGHASH\_FORKID) will remain unaffected by the Chronicle update. For developers aiming to leverage the original protocol's behavior, the Chronicle release offers the option to utilize the Original Transaction Digest Algorithm (OTDA) and the flexibility to determine malleability-related restrictions for transactions.
 
-## Removal of Restrictions
+## Transaction Digest Algorithms and Selective Malleability Restrictions
 
-As mentioned above, in the Chronicle release, malleability rules are being adjusted under the concept of "Relax." These changes depend on the interplay of the `FORKID` and `RELAX` Sighash bits. By default, users who do nothing will retain the current behavior (with `FORKID` active and `RELAX` disabled). To align with the original protocol philosophy, the OTDA digest algorithm will reintroduce relaxed rules where needed. It is also important to mention that It doesn't matter if the transaction configuration involves multi-signatures within a script or across multiple inputs. This approach ensures the protocol integrity while enabling optional adoption of relaxed malleability constraints. The table below describes all possible scenarios and their expected results: \
+As mentioned above, in the Chronicle release, malleability rules are being adjusted under the concept of "Relax." These changes depend on the interplay of the `FORKID` and `RELAX` Sighash bits. By default, users who do nothing will retain the current behavior (with `FORKID` active and `RELAX` disabled). The OTDA will reintroduce relaxed rules where needed. It is also important to mention that it doesn't matter if the transaction configuration involves multiple signatures within a script or across multiple inputs. This approach enables optional adoption of relaxed malleability constraints. The table below describes all possible scenarios and their expected results: \
```

Comment:
You ought to define which of the below rules are considered malleability related. Just so there is a clear delineation.

Malleability related rules:
Min encoding, Low S, No functional opcodes in unlocking, and Cleanstack. 

I think that's it.

Reply:
Also remove the trailing backslash \